src/GitcoinGovernor.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+// forgefmt: disable-start
+import { GovernorCountingSimple, Governor } from "openzeppelin-contracts/governance/extensions/GovernorCountingSimple.sol";
+import { GovernorVotesComp, ERC20VotesComp } from "openzeppelin-contracts/governance/extensions/GovernorVotesComp.sol";
+import { GovernorTimelockCompound, ICompoundTimelock } from "openzeppelin-contracts/governance/extensions/GovernorTimelockCompound.sol";
+import { GovernorSettings } from "openzeppelin-contracts/governance/extensions/GovernorSettings.sol";
+// forgefmt: disable-end
+
+/**
+ * @notice The upgraded Gitcoin Governor—Bravo compatible and built from OpenZeppelin.
+ */
+contract GitcoinGovernor is
+  GovernorCountingSimple,
+  GovernorVotesComp,
+  GovernorTimelockCompound,
+  GovernorSettings
+{
+  /// @notice The address of the existing GTC token on Ethereum mainnet which will interop with
+  /// this Governor after the upgrade is complete.
+  ERC20VotesComp private constant GTC_TOKEN =
+    ERC20VotesComp(0xDe30da39c46104798bB5aA3fe8B9e0e1F348163F);
+
+  /// @notice The address of the existing GitcoinDAO Timelock on Ethereum mainnet which this
+  /// Governor will control after the upgrade is complete.
+  ICompoundTimelock private constant TIMELOCK =
+    ICompoundTimelock(payable(0x57a8865cfB1eCEf7253c27da6B4BC3dAEE5Be518));
+
+  /// @notice Human readable name of this Governor.
+  string private constant GOVERNOR_NAME = "GTC Governor Bravo";
+
+  /// @notice The number of GTC (in "wei") that must participate in a vote for it to meet quorum
+  /// threshold.
+  uint256 private constant QUORUM = 2_500_000_000_000_000_000_000_000; // 2,500,000 GTC
+
+  /**
+   * @param _initialVotingDelay The deployment value for the voting delay this Governor will
+   * enforce.
+   * @param _initialVotingPeriod The deployment value for the voting period this Governor will
+   * enforce.
+   * @param _initialProposalThreshold The deployment value for the number of GTC required to submit
+   * a proposal this Governor will enforce.
+   */
```

Comment:
Per our style guide these should also be triple-slash style natspec

Reply:
I think it's uglier 🙈 

But done in 78a11b444add9c6e0fb59e6751995cb55d6c82dd

Reply:
Don't worry you'll come around to it's superiority soon enough 😛